rfd/0008-application-access.md

```diff
@@ -168,6 +168,27 @@ Below are examples of the three events.
 }
 ```
 
+### FQDN
+
+By default applications are available at `appName.localProxyAddress`. This is the address most users will access an application at. An example would be `grafana.example.com`.
```

Comment:
In other places the RFD uses `proxy.example.com` as the example proxy address; `grafana.proxy.example.com` may be therefore more descriptive. (Also: `acme.proxy.example.com` in the Logout section?)